apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -130,6 +131,20 @@ type Application struct {
 	//
 	// +optional
 	Selector *LabelSelector `json:"selector,omitempty"`
+
+	// Flow control limit to be applied to the aggregated log
```

Comment:
golang comment pattern is to start with the field or method name (e.g. GroupLimit ...)